microservice_demo/meetup_service.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import requests, os, json\n",
+    "ALCHEMY_KEY = os.getenv('ALCHEMY_KEY')\n",
+    "MAKER_KEY = os.getenv('MAKER_KEY')\n",
+    "MAKER_EVENT_NAME = os.getenv('MAKER_EVENT_NAME')\n",
+    "TOPIC = os.getenv('TOPIC', 'bluemix')\n",
+    "\n",
+    "MAKER_EVENT_URL = 'https://maker.ifttt.com/trigger/{event}/with/key/{key}'\n",
+    "MEETUP_URL = 'http://www.meetup.com/find/events/?allMeetups=false&keywords={topic}&radius=50&userFreeform={city}&change=yes&eventFilter=mysugg'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# GET /_echo\n",
+    "print(REQUEST)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "def send_maker_request(username, location, meetup_url):\n",
+    "    post_url = MAKER_EVENT_URL.format(event=MAKER_EVENT_NAME, key=MAKER_KEY)\n",
+    "    values = dict(value1=username, value2=location, value3=meetup_url)\n",
+    "    r = requests.post(post_url, data=json.dumps(values), headers={'Content-Type': 'application/json'}, timeout=90)\n",
+    "    r.raise_for_status()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# GET /_city\n",
+    "'''\n",
+    "Check an arbitrary string against the Alchemy entity extraction, using the set API key\n",
```

Comment:
These are evaled as strings every time the code cell runs. Move the comments to markdown above the cell.